lib/src/find.dart

```diff
@@ -0,0 +1,154 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// fifo.dart
+// Implementation of FIFOs.
+//
+// 2023 March 13
+// Author: Rahul Gautham Putcha <rahul.gautham.putcha@intel.com>
+//
+
+import 'dart:math';
+
+import 'package:rohd/rohd.dart';
+import 'package:rohd_hcl/rohd_hcl.dart';
+import 'package:rohd_hcl/src/utils.dart';
+
+/// Doc coming soon...
+class FindFirstOne extends Module {
+  /// Doc coming soon...
+  late Logic _output;
+
+  /// Doc coming soon...
+  Logic get index => _output;
+
+  /// Doc coming soon...  First 1 from the least significan (right)
+  FindFirstOne(Logic bus) {
```

Comment:
Perhaps all these `Find`s can be collapsed into one class?  Finding the N'th of something can be used to find the 0'th of it, and finding 0's and 1's is pretty similar.